scripts/regenerate_protos.sh

```diff
@@ -18,14 +24,46 @@ $PWD/node_modules/grpc-tools/bin/protoc \
     --grpc_out=$TS_OUT_DIR \
     --grpc_out=$JS_OUT_DIR \
     --plugin=protoc-gen-grpc=`which grpc_tools_node_protoc_plugin` \
-    --proto_path=$PWD/xpring-common-protocol-buffers/proto \
-    $PWD/xpring-common-protocol-buffers/**/*.proto
+    --proto_path $PROTO_PATH \
+    ./rippled/src/ripple/proto/rpc/v1/*.proto
 
 # Generate typescript declaration files.
 $PWD/node_modules/grpc-tools/bin/protoc \
     --plugin=protoc-gen-ts=./node_modules/.bin/protoc-gen-ts \
     --ts_out=$TS_OUT_DIR \
     --ts_out=$JS_OUT_DIR \
+    --proto_path=$PROTO_PATH \
```

Comment:
does this generate classes for ./rippled/src/ripple/proto/ripple.proto too?

Reply:
No, this is the include directory for protocol buffers to look for `imports` in their files.

The protocol buffers that are matched by the wildcard on line 28 and 36, which only matches `.proto` files in the `rpc/v1` directory.